gordo_components/builder/mlflow_utils.py

```diff
@@ -242,29 +242,36 @@ def get_batch_kwargs(metadata: dict) -> dict:
         instances.
     """
 
+    metric_list: List[Metric] = list()
+    build_metadata = metadata["metadata"]["build-metadata"]
+
+    # Projecti/machine parameters
     param_list = get_params(metadata, ["project-name", "name"])
 
+    # Dataset parameters
     dataset_keys = [
         "train_start_date",
         "train_end_date",
         "resolution",
         "filter",
         "row_filter_buffer_size",
     ]
-    model_keys = ["model-creation-date", "model-builder-version", "model-offset"]
-    for section, keys in zip(["dataset", "model"], [dataset_keys, model_keys]):
-        param_list += get_params(metadata[section], keys)
+    param_list += get_params(metadata["dataset"], dataset_keys)
 
-    metric_list: List[Metric] = list()
+    # Model parameters
+    model_keys = ["model-creation-date", "model-builder-version", "model-offset"]
+    param_list += get_params(build_metadata["model"], model_keys)
 
     # Parse cross-validation metrics
-    if metadata["model"].get("cross-validation", {}).get("scores", None):
-        scores = metadata["model"]["cross-validation"]["scores"]
+    tag_list = normalize_sensor_tags(
+        metadata["dataset"]["tag_list"], asset=metadata["dataset"]["tag_list"]
```

Comment:
Looks strange? 
`asset=metadata["dataset"]["tag_list"]` - don't think that makes sense.

Reply:
It would if we wanted to set the `tag_list` to the `asset`, buuuut... I suppose we don't.